tests/random1.py

```diff
@@ -109,6 +110,7 @@ def setUp(self):
     def tearDown(self):
         del self.image
 
+    @unittest.skipIf(sys.version_info.major >= 3, "setState segv on Py3k. Bytes vs str issue.")
```

Comment:
Not on this branch. I will cherry-pick the fix from @TallJimbo tonight. We then have to decide how much we care about the exception problem.